image_rotate/src/image_rotate.cpp

```diff
@@ -38,13 +38,6 @@ int main(int argc, char ** argv)
   // Force flush of the stdout buffer.
   setvbuf(stdout, NULL, _IONBF, BUFSIZ);
 
-  if (argc <= 1) {
-    RCUTILS_LOG_WARN(
-      "Topic 'image' has not been remapped! Typical command-line usage:\n"
-      "\t$ ros2 run image_rotate image_rotate image:=<image topic>");
-    return 1;
-  }
```

Comment:
This debug message didn't work at all - made a more robust version by checking the overlap of our topic name with the actual topics that exist (as many of our depth_image_proc stuff does)